.gitmodules

```diff
@@ -23,10 +23,13 @@
 [submodule "hybris/libhybris"]
 	path = hybris/mw/libhybris
 	url = https://github.com/mer-hybris/libhybris
-	branch = egl_libs
 [submodule "hybris/busybox"]
 	path = hybris/busybox
 	url = https://github.com/mer-hybris/android_external_busybox_prebuilt
 [submodule "hybris/selinux_stubs"]
 	path = hybris/selinux_stubs
 	url = https://github.com/mer-hybris/android_external_selinux_stubs
+[submodule "hybris-patches"]
+	path = hybris-patches
+	url = https://github.com/mer-hybris/hybris-patches
+	branch = hybris-sony-pie
```

Comment:
As per existing branch naming convention in the manifest [1], this should read hybris-sony-aosp-pie
Because historically we default all branches to LineageOS (e.g. `hybris-14.1`), and if something is AOSP, then we specify it (e.g. `hybris-aosp-5.1.0_r5`). Sony keyword is also a special case of patchsets.

[1] https://github.com/mer-hybris/android/branches